src/content/docs/logs/ui-data/parsing.mdx

```diff
@@ -326,6 +326,16 @@ Note that variable names must be explicitly set and be lowercase like `%{URI:uri
             Geographic location from IP addresses. See [Geolocating IP addresses (GeoIP)](#geo) for more information.
           </td>
         </tr>
+
+        <tr>
+          <td>
+            `key value pairs`
```

Comment:
"key value pairs" or "key-value pairs? (there's usages of both hyphenated and non-hyphenated, maybe we should align on one of them -- looks like hyphenation might be the way to go? https://en.wikipedia.org/wiki/Name%E2%80%93value_pair)